targets/TARGET_ARM_SSG/TARGET_MPS2/TARGET_MPS2_M0/device/TOOLCHAIN_ARM_STD/startup_MPS2.S

```diff
@@ -61,6 +49,17 @@ __heap_base
 Heap_Mem        SPACE   Heap_Size
 __heap_limit
 
+; <h> Stack Configuration
+;   <o> Stack Size (in Bytes) <0x0-0xFFFFFFFF:8>
+; </h>
+
+Stack_Size      EQU     0x00001000
```

Comment:
Hi, I think the conclusion from #7238 is "ISR stack should be exactly 1 kB", where ISR stack is this one defined in the startup scripts. This should be 0x400 to pass the proposed tests.
@kapi90 @cmonr

Reply:
Ah, good catch! This is correct, although the build issues that we were seeing appear to be CI related.

For reference, it looks like the pattern is a bit more nuanced than simply setting Stack_Size to 0x400: https://github.com/ARMmbed/mbed-os/pull/7238/commits/7cfa0890f5a6acff06c7507c5b342379bd574ba4#diff-3c01454dfe6f5081adfd7fdde195967fR18

Reply:
yes, if the target support both mbed os 2 and 5, the definitions in the above link are suggested